code/stable/dblpendulum/src/cpp/Calculations.hpp

```diff
@@ -1,6 +1,7 @@
 /** \file Calculations.hpp
     \author Dong Chen
     \brief Provides functions for calculating the outputs
+    \"Generated by Drasil v0.1-alpha"
```

Comment:
Do we want these quotation marks? I personally don't think so

Reply:
I was saying that we should remove the quotation marks in my message. I can see how I phrased my message awkwardly, sorry about that...

Reply:
Yeah I figured the quotation marks in your message were just saying what should be displayed and shouldn't be included in the actual output

Reply:
Ohhh ok! My bad. I will change that right now!